src/bridge/RoomAccessSyncer.ts

```diff
@@ -226,7 +226,7 @@ export class RoomAccessSyncer {
         );
         if (matrixRooms.length === 0) {
             req.log.info("No mapped matrix rooms for IRC channel %s", channel);
-            return;
+            return BridgeRequestErr.ERR_NOT_MAPPED;
```

Comment:
Given the pattern of returning undefined vs an error enum appears elsewhere, I'm confident that this works, but where are these enum values handled?